dotnet/src/Connectors/Connectors.Memory.AzureAISearch/AzureAISearchCollectionCreateMapping.cs

```diff
@@ -121,9 +121,10 @@ public static SearchFieldDataType GetSDKFieldDataType(Type propertyType)
             Type t when t == typeof(bool) => SearchFieldDataType.Boolean,
             Type t when t == typeof(int) => SearchFieldDataType.Int32,
             Type t when t == typeof(long) => SearchFieldDataType.Int64,
+            // We don't map float to SearchFieldDataType.Single, because Azure AI Search doesn't support it.
+            // Half is also listed by the SDK, but currently not supported.
```

Comment:
Source: I implemented the support and then got error from the Azure AI Search service